ghproxy/ghmetrics/ghmetrics.go

```diff
@@ -45,13 +59,18 @@ func CollectGithubTokenMetrics(headers http.Header, now time.Time) {
 		logrus.WithError(err).Warningf("Couldn't convert number of remaining token requests into gauge value (float)")
 	}
 
-	ghTokenUsageGaugeVec.With(prometheus.Labels{"remaining": remaining, "until_reset": timeUntilReset.String()}).Set(remainingFloat)
+	muxTokenUsage.Lock()
+	defer muxTokenUsage.Unlock()
+	ghTokenUntilResetGaugeVec.With(prometheus.Labels{"token_hash": tokenHash, "until_reset": timeUntilReset.String()}).Set(float64(timeUntilReset.Nanoseconds()))
+	ghTokenUsageGaugeVec.With(prometheus.Labels{"token_hash": tokenHash, "remaining": remaining}).Set(remainingFloat)
```

Comment:
We don't need the remaining tokens as a label -- that's the value of the gauge already